build-system/test-server.js

```diff
@@ -111,5 +111,17 @@ app.use('/form/post', function(req, res) {
   });
 });
 
+/**
+ * Serve entry point script url
+ */
+app.get('/dist/ww.js', function(req, res, next) {
```

Comment:
@lanka I've pulled in that PR and I'm still seeing errors while testing if I don't first run `gulp dist`. Without the changes to `test-server.js`, the test still requests the minified web worker.